src/rdkafka_sasl_plain.c

```diff
@@ -70,10 +70,14 @@ int rd_kafka_sasl_plain_client_new(rd_kafka_transport_t *rktrans,
                                    size_t errstr_size) {
         rd_kafka_broker_t *rkb = rktrans->rktrans_rkb;
         rd_kafka_t *rk         = rkb->rkb_rk;
+        rd_kafka_conf_t *conf  = &rk->rk_conf; /* Not using rd_kafka_conf because of const modifier */
```

Comment:
Other places in this function already use rk->rk_conf directly, so no need for this alias.

Reply:
Done